cmd/kube-apiserver/app/aggregator.go

```diff
@@ -263,6 +263,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "batch", Version: "v1"}:                              {group: 17400, version: 15},
 	{Group: "batch", Version: "v1beta1"}:                         {group: 17400, version: 9},
 	{Group: "batch", Version: "v2alpha1"}:                        {group: 17400, version: 9},
+	{Group: "certificates.k8s.io", Version: "v1"}:                {group: 17300, version: 15},
```

Comment:
What is `version` here?

Reply:
priority of this version relative to other versions in the same group... higher priority controls discovery order. we arbitrarily left a few slots available between betas and stable versions and this matches what we did elsewhere

Reply:
So version means priority?  Got it.